collections/hrsi-rlie-s1.yaml

```diff
@@ -12,7 +12,7 @@ EDCBrowser: ""
 EOBrowser: ""
```

Comment:
Could we add EOB link for the rlie collections? I think these collections are available on EOB. If yes, we should add EOB link for `rlie-s1`, `rlie-s1s2` and `rlie-s2`.

Reply:
Sure, added